spec/services/statement_classifier_spec.rb

```diff
@@ -91,6 +91,7 @@
     context 'when statement is actionable' do
       before do
         position_held.group = nil
+        statement.verifications.build(status: true)
```

Comment:
Yeah this is fine. Good call to expand this spec as this is certainly the behaviour we want.